service/commentutil/commentutil.go

```diff
@@ -77,6 +77,35 @@ func MarkdownComment(c *reviewdog.Comment) string {
 	return sb.String()
 }
 
+// MarkdownSuggestions creates diff in markdown for suggested changes
+func MarkdownSuggestions(c *reviewdog.Comment) string {
+	suggestions := c.Result.Diagnostic.GetSuggestions()
+	if len(suggestions) == 0 {
+		return ""
+	}
+
+	var sb strings.Builder
+
+	suggestionsRendered := 0
+	for _, s := range suggestions {
+		if s.Range == nil || s.Range.Start == nil || s.Range.End == nil {
+			continue
+		}
+		if suggestionsRendered > 0 {
+			sb.WriteString("\n\n")
+		}
+		sb.WriteString("```suggestion:-0+")
```

Comment:
Probably it's safer to use 4 backticks if it also works when the suggestion doesn't contain code-fence block? 
https://docs.gitlab.com/ee/user/discussions/#code-block-nested-in-suggestions
Can you check?